### PR TITLE
Update manufactured_parts_quality_information -> update of shared shared BPNS trait

### LINKED ISSUE #626: [Model Update]: Update Fleet.Vehicles -> update of shared shared.business_partner_number to latest version
## Update Reason
The current version of Fleet.Vehicles was released 2 weeks ago. Afterwards we found a regex failure in the shared aspect model shared.business_partner_number that is used in this semantic model (BPNS regex failure).
Update is to change the reference to the lates shared model.

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists Quality Management Use Case
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [x] The potential update will be either 
  - [x] backward compatible or 
  - [ ] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by TorstenHirsch**
### Added
- property added parentSerialNumber

### Changed
- reference to shared aspect model for BPNS was changed to latest version

### Removed

Closes #626 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by TorstenHirsch** [comment]
@agg3fe Could you check the model? Thank you
modelling_team

File: io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.manufactured_parts_quality_information:2.0.0#> .
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl:28]: 2.1.0

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by TorstenHirsch** [comment]
Hi @agg3fe I'm on vacation next week. I ask today, who from CX use case quality could proxy me on Monday's meeting. Change of model was discussed and agreed by CX quality use case team.
Change has no effect of the test data - I checked the already existing test data with changed model.

In case I find nobody could you bring this to MS3 next week?

**Comment by agg3fe** [comment]
> Hi @agg3fe I'm on vacation next week. I ask today, who from CX use case quality could proxy me on Monday's meeting. Change of model was discussed and agreed by CX quality use case team. Change has no effect of the test data - I checked the already existing test data with changed model.
> 
> In case I find nobody could you bring this to MS3 next week?

Ok, I can do that.

File: io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+   samm:characteristic :PositiveNumber ;
+   samm:exampleValue "1"^^xsd:positiveInteger .
+
+:parentPartNumber a samm:Property ;
+   samm:preferredName "Parent part number"@en ;
+   samm:description "Internal part number of the analysed part.\nIf the manufacturer is a TIER1 supplier -> parent level is OEM.\nIf the manufacturer is a TIER2 supplier -> parent level is TIER1 supplier.\nA parent part number is NOT unique: All parent parts of the same kind share the same parent part number."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "12345" .
+
+:parentSerialNumber a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl:129]: property naming changes required: 
`customerPartInstanceId` or
`parentPartInstanceId`

File: io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "GBH3232434535454545-3534535" .
+
+:numberOfConductedEOLTests a samm:Property ;
+   samm:preferredName "Conducted EOL test"@en ;
+   samm:description "Number how often this part went through the EOL test"@en ;
+   samm:characteristic :PositiveNumber ;
+   samm:exampleValue "1"^^xsd:positiveInteger .
+
+:parentPartNumber a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl:123]: `PartId` or `customerPartId`

Comment (jacewski-bosch) [reply io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl:123]: As this model is used in a not Industry Core related use case, we decided to leave this as is for now. In a future version this should be resolved, which will, be guardrailed by a yet to define MS2 requirement. (#642 )

File: io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl (hunk 2 above)
Comment (jacewski-bosch) [reply io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl:129]: As this model is used in a not Industry Core related use case, we decided to leave this as is for now. In a future version this should be resolved, which will, be guardrailed by a yet to define MS2 requirement. (#642 )

File: io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl (hunk 3 above)
Comment (jacewski-bosch) [reply io.catenax.manufactured_parts_quality_information/2.1.0/ManufacturedPartsQualityInformation.ttl:123]: @agg3fe proceed with MS3 approval in next meeting

**Review by jacewski-bosch — APPROVED** [review]

File: io.catenax.manufactured_parts_quality_information/RELEASE_NOTES.md [hunk 4]
```diff
@@ -2,6 +2,16 @@
 All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
+- Version 2.0.0 of model is deprecated. It has a wrong regex for BPNS validation from the old shared model for BPNS trait.
+
+## [2.1.0] - 2024-02-12
```
Comment (agg3fe) [inline io.catenax.manufactured_parts_quality_information/RELEASE_NOTES.md:7]: update the date

Comment (JulianSchererZF) [reply io.catenax.manufactured_parts_quality_information/RELEASE_NOTES.md:7]: I'm not able to do that due to missing rights on this branch

**Pull request merged**